movo_network/movo_network_config.bash

```diff
@@ -14,23 +14,23 @@
 # A remote PC is ROS slave, the default interface is lo (change it to whatever physical interface is connected to movo if working with real hardware), 
```

Comment:
this bash script uses movo1 & movo2 aliases. these do not always exist. dont have to resolve in this pr just something to remember https://github.com/nightingale-project/kinova-movo/issues/21

Reply:
I think this uses $HOSTNAME which should be available independant of network connectivity.

Reply:
yes, $HOSTNAME will exist but won't line 24 fail? " export ROS_MASTER_URI=http://movo2:11311/ "